pyramid_debugtoolbar/panels/request_vars.py

```diff
@@ -6,6 +6,66 @@
 
 _ = lambda x: x
 
+
+# extractable_request_attributes allow us to programmatically pull data
+# the format is...
+#                                 ( attr_, is_show_dict, is_execute_method )
+extractable_request_attributes = (
+                                  ('accept_charset', None, None),
+                                  ('accept_encoding', None, None),
+                                  ('accept_language', None, None),
+                                  ('application_url', None, None),
+                                  ('authenticated_userid', None, None),
+                                  ('authorization', None, None),
+                                  ('cache_control', None, None),
+                                  ('context', None, None),
+                                  ('current_route_path', None, True),
+                                  ('current_route_url', None, True),
```

Comment:
Remove these method calls and support for the `is_execute_method` please. 1) These 2 methods don't work when traversal is used and 2) these values are acquirable without using the methods (`request.url` and `request.path` if I remember right).